equinox/internal/_omega.py

```diff
@@ -29,13 +29,23 @@ class _ω(metaclass=_Metaω):
         ```
 
         This is entirely equivalent to the above.
+
+    !!! edge case
+
+        If if a tuple `(a, b)` is passed then
+        ```
+        ω(a,b).call(fn) = jax.tree_util.tree_map(lambda x,y: fn(y), b, a)
+        ```
+        where b is a prefix PyTree of a. This evaluates a at the nodes corresponding
+        to the leaves of b.
     """
 
-    def __init__(self, value, is_leaf=None):
+    def __init__(self, *value, is_leaf=None):
```

Comment:
I'd suggest making this be slightly less general: `self, value, *, structure=sentinel, is_leaf=None`